### PR TITLE
World basic gen

### PR DESCRIPTION
**Pull request by Trompettesib**

### CODE REVIEW

**Comment by Trompettesib** [comment]
I don't know if in this task I need to generate a ring of chunks around each players

**Review by STMiki — DISMISSED** [review]

File: cubic-server/default/Overworld.cpp [hunk 1]
```diff
@@ -2,6 +2,8 @@
 #include "../world-storage/WorldStorage.hpp"
 #include "../world-storage/ChunkColumn.hpp"
 
+#define NB_SPAWN_CHUNKS 19
```
Comment (huntears) [inline cubic-server/default/Overworld.cpp:5]: Please use a constexpr variable for stuff like this.
It allows for proper typechecking.

**Review by huntears — CHANGES_REQUESTED** [review]

**Comment by Trompettesib** [comment]
Is it ok if I close this pr and delete the branch ?

**Comment by Trompettesib** [comment]
This PR will be closed because another one (updated) has been made

**Pull request closed without merging**